habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -243,6 +243,7 @@ class HrlDefinedSkillConfig(HabitatBaselinesBaseConfig):
     # If true, this willapply the post-conditions of the skill after it
     # terminates.
     apply_postconds: bool = False
+    ignore_grip: bool = False
```

Comment:
what does this do ?

Reply:
Added a comment in default_structured_config.py